### PR TITLE
Set databag name with an attribute

### PR DESCRIPTION
**Pull request by tschuy**
@jordane 

This is required so that we can run both production and staging instrances with Chef, without reusing the items in the databags.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -81,7 +81,9 @@
   action :run
 end
 
-passwords = Chef::EncryptedDataBagItem.load("ganeti_webmgr", "passwords")
+passwords = Chef::EncryptedDataBagItem.load(
+  'ganeti_webmgr',
```
Comment (jordane) [inline recipes/default.rb]: Usually the data bag, not the data bag item, is set as an attribute.

File: recipes/default.rb [hunk 2]
```diff
@@ -81,7 +81,9 @@
   action :run
 end
 
-passwords = Chef::EncryptedDataBagItem.load("ganeti_webmgr", "passwords")
+passwords = Chef::EncryptedDataBagItem.load(
+  node['ganeti_webmgr']['databag']
```
Comment (armiller) [inline recipes/default.rb]: I believe you need a comma here.

**Comment by tschuy** [comment]
@jordane can you review this again? this is the only blocker for deploying GWM 0.11.

**Comment by jordane** [comment]
+1

**Pull request merged**